jekyll.yml

```diff
@@ -77,6 +77,7 @@ mesh_product_name: Kuma
 mesh_product_name_path: kuma
 mesh_namespace: kuma-system
 mesh_cp_name: kuma-control-plane
+mesh_cp_zone_sync_name_prefix: kuma-
```

Comment:
You should add it the `jekyll-dev.yml` too

Reply:
Yes, already added.